hr_payslip_line_manually/models/hr_salary_rule.py

```diff
@@ -0,0 +1,28 @@
+from odoo import fields, models
+
+
+class HrSalaryRule(models.Model):
+    _inherit = "hr.salary.rule"
+
+    analytic_account_id = fields.Many2one(
+        "account.analytic.account", "Analytic Account"
+    )
+    line_manually_model = fields.Selection(
+        [("hr.contract", "Contract"), ("hr.payslip", "Payslip")],
+        string="Salary Rule input from",
+    )
+
+    def _compute_rule_python(self, localdict):
+        if "result_list" in localdict:
+            # Return list of values dictionary. Each dictionary will be a payslip line.
+            result_list = []
+            for result_dict in localdict["result_list"]:
+                values = super(HrSalaryRule, self)._compute_rule_python(result_dict)
+                values["analytic_account_id"]: result_dict.get("result_analytic")
+                result_list.append(values)
+            return result_list
+        return super(HrSalaryRule, self)._compute_rule_python(localdict)
```

Comment:
Not sure but this will generate a loop. You call super two times in the function.